kedro/framework/cli/starters.py

```diff
@@ -363,6 +363,15 @@ def _create_project(template_path: str, cookiecutter_args: dict[str, Any]):
     python_package = extra_context.get(
         "python_package", project_name.lower().replace(" ", "_").replace("-", "_")
     )
+    add_ons = extra_context.get("add_ons")
+
+    # Only non-starter projects have configurable add-ons
+    if template_path == str(TEMPLATE_PATH):
+        if add_ons == "none":
+            click.secho("\nYou have selected no add-ons")
+        else:
+            click.secho(f"\nYou have selected the following add-ons: '{add_ons}'")
```

Comment:
I noticed that this prints something like "You have selected the following add-ons: '1,4'". Would it be possible to instead print the name of the add-on, so linting and documentation in this case?

Reply:
Added in [e3c7de5](https://github.com/kedro-org/kedro/pull/2987/commits/e3c7de548deb3ec2764f732d586a21ea97dda100) - though maybe a bit clunky? Any thoughts on the implementation?

Reply:
I think this is fine 👍